conformance/lang/expressions/shift-expression/left_shift.balt

```diff
@@ -0,0 +1,3816 @@
+             operand.
+Labels: byte, int:Signed8, shift-expr
+
+function errorFunction(int:Signed8 x, int:Signed8 y, byte a) {
+    int:Signed8 _ = x << a; // @error expected 'int:Signed8' but found 'int'
+    byte _ = y << a; // @error expected 'byte' but found 'int'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Signed8' as the LHS operand and user-defined subtype
+             of integer as the RHS operand.
+Labels: DecimalNumber, int:Signed8, shift-expr, type, unary-minus, union-type
+
+type Ints -1|2|-128;
+
+function errorFunction(int:Signed8 x, Ints a) {
+    int:Signed8 _ = x << a; // @error expected 'int:Signed8' but found 'int'
+    Ints _ = x << a; // @error expected 'Ints' but found 'int'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Signed8' as the LHS operand and optional 'int' as
+             the RHS operand.
+Labels: int, int:Signed8, optional-type, shift-expr
+
+function errorFunction(int:Signed8 x, int? a) {
+    int:Signed8? _ = x << a; // @error  expected 'int:Signed8?' but found 'int?'
+    int _ = x << a; // @error expected 'int' but found 'int?'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Signed8' as the LHS operand and optional builtin
+             subtypes of integer as the RHS operand.
+Labels: decimal, float, int, int:Signed16, int:Signed32, int:Signed8, int:Unsigned16, int:Unsigned32, int:Unsigned8,
+        shift-expr, optional-type
+
+function errorFunction(int:Signed8 a, int:Signed8? b, int:Unsigned8? c, int:Signed16? d, int:Unsigned16? e,
+                        int:Signed32? f, int:Unsigned32? g) {
+    int:Signed16 _ = a << b; // @error expected 'int:Signed16' but found 'int?'
+    float _ = a << c; // @error expected 'float' but found 'int?'
+    int:Unsigned16 _ = a << d; // @error expected 'int:Unsigned16' but found 'int?'
+    int:Signed16 _ = a << e; // @error expected 'int:Signed16' but found 'int?'
+    decimal _ = a << f; // @error expected 'decimal' but found 'int?'
+    int:Unsigned16 _ = a << g; // @error expected 'int:Unsigned16' but found 'int?'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Signed8' as the LHS operand and optional byte as
+             the RHS operand.
+Labels: byte, int, int:Signed8, optional-type, shift-expr
+
+function errorFunction(int:Signed8 x, byte? a) {
+    int:Signed8 _ = x << a; // @error  expected 'int:Signed8' but found 'int?'
+    int _ = x << a; // @error expected 'int' but found 'int?'
+    byte _ = x << a; // @error expected 'byte' but found 'int?'
+}
+
+Test-Case: output
+Description: Test '<<' expression with 'int:Signed8' operands in function call and method call.
+Labels: function-call-expr, function-defn, int:Signed8, method-call-expr, shift-expr, unary-minus, value:toBalString,
+        value:toString
```

Comment:
Do we need both `function-call-expr` and `method-call-expr` labels?

Reply:
Yeah. Better to add that label too. Thanks for point it out.